components/common/infinite_scroll.jsx

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import debounce from 'lodash/debounce';
+
+import LoadingSpinner from 'components/widgets/loading/loading_spinner.jsx';
```

Comment:
Can you use `import LoadingScreen from 'components/loading_screen.jsx';` component so it feels the same way as other instances in the code.

Reply:
@sudheerDev alright on it.

Reply:
<img width="846" alt="Screen Shot 2019-09-23 at 19 29 13" src="https://user-images.githubusercontent.com/28563179/65444349-899fd900-de38-11e9-9c2d-1cd26119f8f6.png">
Hi @sudheerDev , I have transferred the loader to be part of the list, and it's looking like that currently. Is that aesthetically alright?

Reply:
Just `...` is good enough.You can drop the `Loading` text. we want to keep that as close to other cases such as center channel or search results infinite loader as possible. Though we might add `Loading` optional based on prop we don't need that ATM. We can cross that bridge when we have to.

Reply:
@sudheerDev I implemented this. Let me know if you've some more changes you want in regards to this.